MFVNeutralinoFormats/interface/LeptonVertexAssociation.h

```diff
@@ -0,0 +1,22 @@
+#ifndef JMTucker_MFVNeutralinoFormats_LeptonVertexAssociation_h
+#define JMTucker_MFVNeutralinoFormats_LeptonVertexAssociation_h
+
+#include "DataFormats/Common/interface/AssociationMap.h"
+#include "DataFormats/VertexReco/interface/VertexFwd.h"
+#include "DataFormats/PatCandidates/interface/Muon.h"
+#include "DataFormats/PatCandidates/interface/Electron.h"
+#include "TrackingTools/IPTools/interface/IPTools.h"
+namespace mfv {
+  typedef edm::AssociationMap<edm::OneToMany<reco::VertexCollection, pat::MuonCollection> > MuonVertexAssociation;
+
+  enum { NMuonsByUse, MByDr=NMuonsByUse, NMuonsBy };
+  static const char* muonsby_name __attribute__((used)) = "MByDr";
+
+  typedef edm::AssociationMap<edm::OneToMany<reco::VertexCollection, pat::ElectronCollection> > ElectronVertexAssociation;
+
+  enum { NEleByUse, EByDr=NEleByUse, NEleBy };
+  static const char* electronsby_name __attribute__((used)) = "EByDr";
```

Comment:
@abbywarden I might just be confused, but why have an enum where the second entry is set equal to the first one? Then { NMuonsByUse, MByDr, NMuonsBy } is equivalent to { 0, 0, 1 }. It might be fine, but jumped out to me as a surprise. Maybe those first two entries are intended to always be equal...